### PR TITLE
[Tests-Only] Refactor webui scenarios with 'tries to'

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

This PR
1. Refactors webUI scenarios with "tries to copy" & "tries to move" steps. 
2. Removes unused functions that was previously used in "tries to" step

## Related Issue

- https://github.com/owncloud/web/issues/4680

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

**Review by Talank — COMMENTED** [review]
LGTM

File: tests/acceptance/features/webUIFiles/copy.feature [hunk 1]
```diff
@@ -72,21 +72,24 @@ Feature: copy files and folders
     Given user "Brian" has been created with default attributes
     And user "Brian" has shared folder "simple-folder" with user "Alice" with "read" permissions
     And user "Alice" has logged in using the webUI
-    When the user tries to copy file "lorem.txt" into folder "simple-folder (2)" using the webUI
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFiles/copy.feature:75]: I think we were supposed to keep this line as it is and make changes in the function that this step has been using.

**Comment by kiranparajuli589** [comment]
lgtm :+1:

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by individual-it — CHANGES_REQUESTED** [review]
I think we have still a misunderstanding here
I actually thought we we still use
`When the user tries to ...`
but that step would call `copyResource` etc and ignore any exeptions

**Review by jasson99 — COMMENTED** [review]
IMO, we planned to use tries to with changes in its implementaion.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14061/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14061/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Files</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14061/

<p>












</p></details>

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 2]
```diff
@@ -1225,7 +1233,15 @@ When('the user tries to copy file/folder {string} into folder {string} using the
...
 ) {
-  return client.page.FilesPageElement.filesList().attemptToCopyResource(resource, target)
+  return client.page.FilesPageElement.filesList()
+    .copyResource(resource, target)
+    .catch(error => {
+      if (error.message === 'ElementPresentError') {
+        // do nothing
+      } else {
+        throw error
+      }
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/filesContext.js:1243]:
```suggestion
      if (error.message !== 'ElementPresentError') {
     {
        throw error
      }
```

same above

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 3]
```diff
@@ -1225,7 +1233,15 @@ When('the user tries to copy file/folder {string} into folder {string} using the
...
   target
 ) {
-  return client.page.FilesPageElement.filesList().attemptToCopyResource(resource, target)
+  return client.page.FilesPageElement.filesList()
+    .copyResource(resource, target)
+    .catch(error => {
+      if (error.message === 'ElementPresentError') {
+        // do nothing
+      } else {
+        throw error
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/filesContext.js:1242]: some more information in the error would be good, or?

File: tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature [hunk 4]
```diff
@@ -90,6 +90,7 @@ Feature: move files
     And user "Alice" has logged in using the webUI
     When the user tries to move file "lorem.txt" into folder "simple-folder (2)" using the webUI
     Then it should not be possible to paste files into the current folder using the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature:92]: do these kind of steps do anything useful?

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature (hunk 4 above)
Comment (HariBhandari07) [reply tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature]: It just checks if the confirm button is disabled and nothing useful. So removed.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14134/

<p>




</p></details>

**Review by individual-it — APPROVED** [review]

**Pull request merged**